geminimd/config.py

```
@@ -0,0 +1,82 @@
+
+def load_config() -> Dict[str, Any]:
+    cfg = DEFAULTS.copy()
+    conf_path = find_conf_path()
+    # Load TOML if available
+    try:
+        if conf_path.exists() and tomllib is not None:
+            data = tomllib.loads(conf_path.read_text(encoding="utf-8"))
+            if isinstance(data, dict):
+                cfg.update(data)
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)